vic/drivers/classic/src/read_vegparam.c

```diff
@@ -462,10 +461,10 @@ read_vegparam(FILE  *vegparam,
     }
 
     // Bare soil tile
```

Comment:
Did you notice that this section was in the classic driver where we have not modified the veg_con behavior to include an explicit bare soil tile.

See issue https://github.com/UW-Hydro/VIC/issues/385